tests/freqai/test_freqai_datakitchen.py

```diff
@@ -71,22 +71,22 @@ def test_use_DBSCAN_to_remove_outliers(mocker, freqai_conf, caplog):
     freqai = make_data_dictionary(mocker, freqai_conf)
     # freqai_conf['freqai']['feature_parameters'].update({"outlier_protection_percentage": 1})
     freqai.dk.use_DBSCAN_to_remove_outliers(predict=False)
-    assert log_has_re(r"DBSCAN found eps of 2\.3\d\.", caplog)
+    assert log_has_re(r"DBSCAN found eps of 1.75", caplog)
```

Comment:
i think this should be kept at ".1" precision (also keep in mind that `"."` in regex matches "everything" - so the above would match `1.75` - as well as `1075` or `1s75`).

I've had to reduce this to have the test pass on aarch64 (tests not running in CI ATM)

Reply:
Resolved in #7538